.github/workflows/gh-workflow.yml

```diff
@@ -110,8 +110,9 @@ jobs:
           tags: ${{ steps.meta_v.outputs.tags }}
           labels: ${{ steps.meta_v.outputs.labels }}
       - name: Install Helm
-        run: |
-          curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+        uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Oops, this was meant for a separate PR.. Anyway its just a bit cleaner way to install helm on github actions (previous solution did fail intermittently)